tasks/Module_2/task_1.py

```diff
@@ -0,0 +1,47 @@
+"""Task 1 and 2 in module 2."""
+
+"""Constants and filters/"""
+MAX_INVENTORY_CAP = 80
+HEAVY_WEIGHT_THRESHOLD = 70
+LIGHT_WEIGHT_THRESHOLD=60
+TRASH = ('rubbish', 'chewed gum', 'used tissue')
+
+
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+inv = {'gold coin': 42, 'rope': 1}
+dragonLoot = [
```

Comment:
I know that our book recommends this convention; however, it's not pythonic.
We should stick to PascalCase for class names, UPPER_CASE for constants, and snake_case for all the rest.

You can read more about naming conventions here:
https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html

Please convert it to `snake_case`

Reply:
Ok